src-relution/templates/mwFileUpload/mwFileUpload.html

```diff
@@ -14,7 +14,7 @@
           <div class="progress" ng-style="{width: uploadProgress + '%'}"></div>
         </div>
       </div>
-      <div class="btn btn-danger" ng-click="abort()">{{'rlnUikit.mwFileUpload.abort' | i18n}}</div>
+      <div ng-if="showCancelButton" class="btn btn-danger" ng-click="abort()">{{'rlnUikit.mwFileUpload.abort' | i18n}}</div>
```

Comment:
It's better to use the attribute hideCancelButton, so it is visible by default and can be hidden. In most cases a cancel button makes sense, so the default case should be visible. There are only a few cases where you don't want to see the cancel button